react-live.css

```diff
@@ -92,3 +92,9 @@
 .token.deleted {
   color: red;
 }
+
+.tabGuarded{
```

Comment:
Btw it'll also need to be added here: https://raw.githubusercontent.com/FormidableLabs/react-live/master/src/constants/css.js

Sorry for that not being in one file right now :(